packages/alias/README.md

```diff
@@ -27,6 +27,8 @@ import batman from 'batman';
 
 If this seems familiar to Webpack users, it should. This is plugin mimics the `resolve.extensions` and `resolve.alias` functionality in Webpack.
 
+This plugin is working for all kind of files (.ts, .svelte, ...)
```

Comment:
What was the intent behind adding this line?

Reply:
I thought this plugin is not compatible with ts-files but this plugin makes the typescript paths feature (https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) kind of obsolet.

Reply:
```suggestion
This plugin will work for any file type that Rollup natively supports, or those which are [supported by third-party plugins](https://github.com/rollup/awesome#other-file-imports).
```